source/azure_iot_internal.c

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License. */
+
+#include "internal/azure_iot_internal.h"
+
+#include "azure_iot_result.h"
+#include "azure/az_core.h"
+
+AzureIoTResult_t _AzureIoT_TranslateCoreError( az_result xCoreError )
+{
+    AzureIoTResult_t xResult;
+
+    switch( xCoreError )
+    {
+        case AZ_OK:
+            xResult = eAzureIoTSuccess;
```

Comment:
Right now all these errors are randomly placed and hard to review which errors we are missing to translate. Is it possible to either group using comments ? something like 


 // === JSON error codes ===


  // === HTTP Adapter error codes ===